twisted/web/test/test_http2.py

```diff
@@ -1403,6 +1403,32 @@ def validate(streamID):
         return a._streamCleanupCallbacks[1].addCallback(validate)
 
 
+    def test_resetAfterBody(self):
+        """
+        A client that immediately resets after sending the body causes Twisted
+        to send no response.
+        """
+        f = FrameFactory()
+        transport = StringTransport()
+        a = H2Connection()
+        a.requestFactory = DummyHTTPHandler
+
+        requestBytes = f.preamble()
```

Comment:
My problem with this DSL is that the RFC does not contain the 'preamble' name.
So if I got ot the RFC and the search for preamble I got nothing :)

From what I can see, in the RFC this is called `the connection preface` and `f.preamble()` looks like `f.clientConnectionPreface()`

For a DSL I am expecting to see the same names as in the RFC, so that someone can search the RFC and see what is going on there